ClemBot.Bot/bot/custom_tag_prefix.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+import bot.bot_secrets as bot_secrets
+from bot.clem_bot import ClemBot
+from bot.errors import TagPrefixRequestError
+
+log = logging.getLogger(__name__)
+
+
+class CustomTagPrefix:
+
+    def __init__(self, *, default: str):
+        log.info(f'Setting default prefix too: "{default}""')
```

Comment:
Should probably change the log message to specify that's it's the default *tag* prefix

Reply:
Fixed it. There were so many prompt messages to change that I forgot this.